crates/bevy_gizmos/src/circles.rs

```diff
@@ -0,0 +1,142 @@
+//! Additional [`Gizmo`] Functions -- Arrows
+
+use crate::prelude::Gizmos;
+use bevy_math::{Quat, Vec2, Vec3};
+use bevy_render::color::Color;
+use std::f32::consts::TAU;
+
+pub const DEFAULT_CIRCLE_SEGMENTS: usize = 32;
```

Comment:
I'm fine with this getting exposed in this PR, but if so, doc strings.

Reply:
Yeah i thought it was pub before. just reverted it; that can be a different PR.